G/GPMaxlik/Package.toml

```diff
@@ -0,0 +1,3 @@
+name = "GPMaxlik"
+uuid = "988d40dc-a58a-4803-bd2c-6d7438fe27fe"
+repo = "git@git.sr.ht:~cgeoga/GPMaxlik.jl"
```

Comment:
This could be the https URL for consistency (if supported).

Reply:
Makes sense, thank you. I've changed it to the HTTPS url.